v2/internal/pkg/clusterresources/clusterresources.go

```diff
@@ -61,6 +61,10 @@ const (
 )
 
 func (o *ClusterResourcesGenerator) IDMS_ITMSGenerator(allRelatedImages []v2alpha1.CopyImageSchema, forceRepositoryScope bool) error {
+	if len(allRelatedImages) == 0 {
+		o.Log.Info("📄 Nothing mirrored. Skipping IDMS and ITMS files generation.")
+		return nil
+	}
 	o.Log.Info("📄 Generating IDMS and ITMS files...")
```

Comment:
In case we have specific messages for both ITMS and IDMS, this line is not necessary anymore.

Reply:
Removed